### PR TITLE
Upgrade GKE Autopilot instructions

### PR DESCRIPTION
**Pull request by mangulonr**
Please follow [conventional commit standards](https://www.conventionalcommits.org/en/v1.0.0/)
in your commit messages and pull request title.

## Give us some context

* What problems does this PR solve?
* Add any context that will help us review your changes such as testing notes,
  links to related docs, screenshots, etc.
* If your issue relates to an existing GitHub issue, please link to it.

### CODE REVIEW

**Comment by akristen** [comment]
Hi @mangulonr ! If you could sign the CLA, we'll go ahead and get this merged. Thanks a bunch!

**Review by csongnr — APPROVED** [review]

**Comment by mangulonr** [comment]
> Hi @mangulonr ! If you could sign the CLA, we'll go ahead and get this merged. Per our SLAs, this will be reviewed + merged within the next 3 days! I'll follow up with any questions as they come up in review. Thanks a bunch for submitting this PR!

Done!

File: src/content/docs/kubernetes-pixie/kubernetes-integration/installation/kubernetes-integration-install-configure.mdx [hunk 1]
```diff
@@ -1367,9 +1367,11 @@ There are three ways to install the Kubernetes integration in your GKE Autopilot
         title="Guided install (recommended)"
     >
 
-Follow the instructions in the **Guided install** tab above. 
+Follow the instructions in the **Guided install** tab above where you can choose any of these 3 options. 
 
-The guided install will ask if you are installing in a GKE Autopilot cluster. After you enter `Y`, we will set `privileged`, `controlPlane`, `logging`, and `pixie` to `false`. The install will also set the kubelet scheme to http and kubelet port to 10255. 
+* New Relic CLI
```
Comment (akristen) [inline src/content/docs/kubernetes-pixie/kubernetes-integration/installation/kubernetes-integration-install-configure.mdx:1372]: This is a little confusing. My understanding is that they can install GKE autopilot using the same guided install instructions as seen in the first tab of this doc (Guided install (recommended)). That makes sense. I'm not sure how the 3 bullets fit into it. 

How do the bullet points for Helm command and manifest found in the guided collapser differ from the collapsers in this section? Should the bulleted list in the guided install collapser be broken out to the introduction? For example:

> There are three ways to install the Kubernetes integration in your GKE Autopilot cluster.
> 
> * The New Relic CLI found in our guided install
> * A Helm command with pre-populated required values
> * A plain manifest with pre-populated required values

Then the collapser titled guided install would just read:

> You can install in GKE autopilot using the same install flow as our [guided install](/docs/kubernetes-pixie/kubernetes-integration/installation/kubernetes-integration-install-configure/#guided-install) tab.

**Comment by akristen** [comment]
@mangulonr I made a commit of recommended copy changes. See my previous comment about why i broke out that bulleted list into the GKE intro. If i misunderstood what the original content said, LMK so we can set up a call. :) If it's good, also LMK and I can get this approved and merged

**Review by akristen — APPROVED** [review]
Looks good!

**Pull request merged**